tests/unit/Makefile.am

```diff
@@ -16,12 +17,14 @@ endif
 TESTS = $(check_SCRIPTS)
 
 EXTRA_DIST = \
+	harness_freebsd_ports \
 	mock_apt_get \
 	mock_dpkg \
 	mock_dpkg_deb \
 	mock_dpkg_query \
 	mock_pkgin \
 	mock_pkg_info \
+	mock_pkg \
```

Comment:
This should be `mock_freebsd_pkg` to match the file name. `make dist` fails without this.